app/src/androidTest/java/com/github/sdp/ratemyepfl/database/FakeRestaurantRepository.kt

```diff
@@ -8,9 +8,9 @@ class FakeRestaurantRepository @Inject constructor() : RestaurantRepositoryInter
 
     companion object {
         val RESTAURANT_LIST = listOf(
-            Restaurant(id = "Roulotte du Soleil", 0, 1.0, 2.0, 15, 2.5),
-            Restaurant(id = "Arcadie", 0, 3.0, 4.0, 15, 2.5),
-            Restaurant(id = "Takinoa", 0, 5.0, 6.0, 15, 2.5)
+            Restaurant(id = "Roulotte du Soleil", 15, 2.5, 46.519214, 6.567553),
```

Comment:
This will not work, you must either specify the arguments or add an occupancy value

Reply:
Yes I have just noticed that, it's fixed now :)